p2p/proto/transaction.proto

```diff
@@ -0,0 +1,65 @@
+syntax = "proto3";
+import "p2p/proto/common.proto";
+
+message TransactionCommon {
+    Felt252 nonce = 1; // FFU: volition will also add the selected state tree
+    Felt252 version = 2;
```

Comment:
dont we need signature as well?

Reply:
The idea was you verify transactions belong to the block by matching the commitment in the header.